frontend/src/components/Dashboard/SideBar.js

```diff
@@ -0,0 +1,261 @@
+                              ],
+                          },
+                ),
+                contentType: 'application/json; charset=utf-8',
+                success: (data) => {
+                    if (stateIndex < states.length) {
+                        bundlesDict[states[stateIndex]] = data.response.result;
+                        fetchBundles(stateIndex + 1, bundlesDict);
+                    } else {
+                        bundlesDict['floating'] = data.response.result;
+                        const bundles: HTMLElement[] = [];
+                        for (let state in bundlesDict) {
+                            // Only show the non-zero items
+                            if (bundlesDict[state] > 0) {
+                                bundles.push(
+                                    <li key={state} className={classes.subheader}>
+                                        <a
+                                            className={classes.subheader}
+                                            href={
+                                                'https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/bundle_state.py'
+                                            }
+                                        >
+                                            {state}
+                                        </a>
+                                        {': ' + bundlesDict[state]}
+                                    </li>,
+                                );
+                            }
+                        }
+                        this.setState({ bundles: bundles });
+                    }
+                },
+                error: (xhr, status, err) => {
+                    console.error(xhr.responseText);
+                },
+            });
+        };
+        // Start to fetch the bundles' count
+        fetchBundles(0, {});
+    }
+
+    render() {
+        const { classes, userInfo } = this.props;
+        if (!userInfo) {
+            return null;
+        }
+        return (
+            <Card className={cx(classes.card)} elevation={0} style={{ height: '100%' }}>
+                <Avatar src={'https://i.pravatar.cc/30'} className={classes.avatar} />
+                <Box className={classes.box}>
+                    <h3 className={classes.name}>{userInfo.user_name}</h3>
+                    {userInfo.affiliation ? (
+                        <p className={classes.affiliation}>Affiliation: {userInfo.affiliation}</p>
+                    ) : null}
+                </Box>
+                <Divider />
+
+                {this.props.showQuota ? (
+                    <Box className={classes.box}>
+                        <a
+                            className={classes.subheader}
+                            href={
+                                'https://github.com/codalab/codalab-worksheets/blob/master/docs/FAQ.md#how-do-i-reduce-the-amount-of-disk-usage'
```

Comment:
Can you link to the corresponding section on the actual docs site (https://codalab-worksheets.readthedocs.io/)?

Reply:
Thanks for pointing it out. Done